week-2/Homework/mandatory/2-function-creation.js

```diff
@@ -5,7 +5,22 @@ Write a function that:
 - removes any forward slashes (/) in the strings
 - makes the string all lowercase
 */
-function tidyUpString(strArr) {}
+console.clear(); // clean all outputs in console and when you run the code you'll see only last output
+
+function tidyUpString(strArr) {
+  var newArr = strArr;
+  for (let i = 0; i < newArr.length; i++) { // this is loop for reed all items in arry 
+    newArr[i] = newArr[i].trim(); // return items in arry without any space at the begining and end of strings!
+    var container = ""; // empety container for keep arry items without "/"!
+    for (let j = 0; j < newArr[i].length; j++) { // this is a loop for check all char in all items in arry!
+      if (newArr[i].charAt(j) !== "/") {
+        container += newArr[i].charAt(j); // if char is not equal "/" so add to container!
+      }
+    }
+    newArr[i] = container.toLocaleLowerCase(); // replace old value of newArr[i] to new value without any space at the begining and end of string and without any "/"! 
+  }
+  return newArr;
+}
```

Comment:
Good work. There is also a method called replace() that makes this easier, you'll learn about this on Sunday.